File: EtiveMor.OpenImoCiiCalculator/EtiveMor.OpenImoCiiCalculator.Core/Models/Dto/FuelTypeConsumption.cs
using EtiveMor.OpenImoCiiCalculator.Core.Models.Enums;

namespace EtiveMor.OpenImoCiiCalculator.Core.Models.Dto
{
    /// <summary>
    /// A request object to hold metadata about the fuel consumption of a ship
    /// </summary>
    public class FuelTypeConsumption
    {
        /// <summary>
        /// The type of fuel consumed by the ship
        /// </summary>
        public TypeOfFuel FuelType { get; set; }

        /// <summary>
        /// The amount of fuel consumed by the ship in the given year (in grams)
        /// </summary>
        public double FuelConsumption { get; set; }
    }

}

File: python-impl/open_imo_cii_calculator/models/dto/fuel_type_consumption.py
from open_imo_cii_calculator.models.fuel_type import TypeOfFuel

class FuelTypeConsumption:
    """
    A request object to hold metadata about the fuel consumption of a ship
    """
    
    def __init__(self, fuel_type: TypeOfFuel, fuel_consumption: float):
        """
        Initialize a fuel type consumption instance
        
        Args:
            - fuel_type: The type of fuel consumed by the ship
            - fuel_consumption: The amount of fuel consumed by the ship (in grams)
        """
        self.fuel_type = fuel_type
        self.fuel_consumption = fuel_consumption